assets/sass/1_basics/_buttons.scss

```diff
@@ -134,6 +134,11 @@ input[type="button"] {
     }
 
 }
+
+.button-right {
+    float: right;
+}
+
 .button-full-width {
   width: 100%;
 }
```

Comment:
I don't think these should live in `buttons.scss`. They are not specific to buttons. I think you could create two `_helpers.scss` classes instead and use throughout the app.

something like:
`.full-width` and `.right`

Reply:
As it is now, part of this new toolbar code is specifically tied to `layout-d`, but I assume we are replacing it throughout the whole PL. If so, then I suggest we go ahead and do that in this branch and PR.